.github/workflows/ci.yml

```diff
@@ -135,10 +135,10 @@ jobs:
 
       - name: Build and test
         run: |
-          ./integration-testing/config-local-host.sh
+          ./tests/local_host/config-local-host.sh
           python3 -c "from urllib.request import urlretrieve; urlretrieve('$BUILDER_HOST/$BUILDER_SOURCE/$BUILDER_VERSION/builder.pyz', 'builder.pyz')"
-          python3 builder.pyz build -p aws-c-http --cmake-extra=-DENABLE_LOCAL_HOST_INTEGRATION_TESTS=ON
-      - uses: actions/upload-artifact@v2
+          python3 builder.pyz build -p aws-c-http --cmake-extra=-DENABLE_LOCALHOST_INTEGRATION_TESTS=ON
```

Comment:
trivial, but it bugs me, for consistency please replace all the "local host" and "local-host" and "local_host" with just "localhost"